src/hostile_eit/util.py

```diff
@@ -33,7 +33,7 @@ def choose_default_thread_count(cpu_count: int) -> int:
 CACHE_DIR = (
     Path(os.environ.get("HOSTILE_CACHE_DIR", ""))
     if os.environ.get("HOSTILE_CACHE_DIR")
-    else Path(user_data_dir("hostile-eit", "Bede Constantinides"))
+    else Path(user_data_dir("hostile_eit-eit", "Bede Constantinides"))
```

Comment:
Could I ask why we need to change this?

Reply:
Oh we definitely don't need this one, that's a bug!

Reply:
Fixed